test/test_files/package_out_02.nt

```diff
@@ -0,0 +1,9 @@
+
+<https://example.org/MyPackage/package> <http://sbols.org/SEP054#conversion> "false"^^<http://www.w3.org/2001/XMLSchema#boolean> .
+<https://example.org/MyPackage/package> <http://sbols.org/SEP054#subPackage> <https://example.org/MyPackage/package> .
```

Comment:
Shouldn't have itself as a sub-package. Instead, it should have all of the same contents as package_out_01.nt, but with the addition of the two subpackages.